modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml

```
@@ -230,26 +267,33 @@
         plk_create_group_if_not_exists__d8_upmeter_resources_garbage: "D8UpmeterProbeGarbage,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
         plk_grouped_by__d8_upmeter_resources_garbage: "D8UpmeterProbeGarbage,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
         plk_labels_as_annotations: "deployment"
-        summary: Garbage produced by controller-manager probe is not being cleaned.
+        summary: Garbage Deployments from the controller-manager probe are not being cleaned up.
         description: |
-          Average probe deployments count per upmeter-agent pod: {{ $value }}.
+          The average number of probe-generated Deployments per `upmeter-agent` Pod is {{ $value }}.
+
+          `upmeter-agent` should automatically delete Deployments created by the `control-plane/controller-manager` probe.
+          There should not be more Deployments than master nodes (since `upmeter-agent` runs as a DaemonSet with a master `nodeSelector`), and they should be deleted within seconds.
 
-          Upmeter agents should clean Deployments produced by control-plane/controller-manager probe. There should not
-          be more deployments than master nodes (upmeter-agent is a DaemonSet with master nodeSelector).
-          Also, they should be deleted within seconds.
+          This may indicate:
 
-          This might be an indication of a problem with kube-apiserver. Or, possibly, the deployments were left by old
-          upmeter-agent pods due to Upmeter update.
+          - A problem with the `kube-apiserver`.
+          - Stale Deployments left by outdated `upmeter-agent` Pods after an Upmeter update.
+          
+          Troubleshooting steps:
 
-          1. Check upmeter-agent logs
+          1. Check the `upmeter-agent` logs:
 
-          `kubectl -n d8-upmeter logs -l app=upmeter-agent --tail=-1 | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "controller-manager") | [.time, .level, .msg] | @tsv'`
+             ```shell
+             kubectl -n d8-upmeter logs -l app=upmeter-agent --tail=-1 | jq -rR 'fromjson? | select(.group=="control-plane" and .probe == "controller-manager") | [.time, .level, .msg] | @tsv'
+             ```
 
-          2. Check that control plane is functional, kube-controller-manager in particular.
+          2. Ensure the control plane is operating normally. Pay close attention to `kube-controller-manager`.
 
-          3. Delete deployments manually:
+          3. Delete stale Deployments manually:
 
-          `kubectl -n d8-upmeter delete deploy -l heritage=upmeter`
+             ```shell
+             kubectl -n d8-upmeter delete deploy -l heritage=upmeter
```

It is suggested to specify full commands everywhere, not aliases. 

```suggestion
             kubectl -n d8-upmeter delete deployment -l heritage=upmeter
```